content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation
```

Comment:
Missing period is preventing the preview (https://app.netlify.com/sites/preliminary-istio/deploys/5ea6e54d7b751300067079ce) from generating.
```suggestion
description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation.
```

Reply:
:+1: